openfe/protocols/openmm_rfe/equil_rfe_settings.py

```diff
@@ -519,19 +424,35 @@ class RelativeHybridTopologyProtocolSettings(Settings):
     class Config:
         arbitrary_types_allowed = True
 
+    # Inherited things
+
+    forcefield_settings: OpenMMSystemGeneratorFFSettings
+    """Parameters to set up the force field with OpenMM Force Fields."""
+    thermo_settings: ThermoSettings
+    """Settings for thermodynamic parameters."""
+
     # Things for creating the systems
     system_settings: SystemSettings
+    """Simulation system settings including the long-range non-bonded method."""
     solvation_settings: SolvationSettings
+    """Settings for solvating the system."""
 
     # Alchemical settings
     alchemical_settings: AlchemicalSettings
+    """Alchemical protocol settings including lambda windows and soft cores."""
```

Comment:
```suggestion
    """Alchemical protocol settings including lambda windows and soft core scaling choices."""
```
It's not a major change just a tiny clarification.